### PR TITLE
Fix pathing issue to installer directory

### PR DESCRIPTION
**Pull request by btoll**
## Summary

Regardless of the channel, the installer scripts are either located at `algorand` or `algorand-devtools` and DO NOT include the channel name (hence, the package name shouldn't be used to construct the path).

## Test Plan

Jenkins job.

### CODE REVIEW

**Review by onetechnical — APPROVED** [review]

**Review by onetechnical — APPROVED** [review]

File: scripts/release/build/deb/package.sh [hunk 1]
```diff
@@ -27,7 +27,13 @@ for pkg_name in "${PKG_NAMES[@]}"; do
         exit 1
     fi
 
-    cp -p "${DEBTMP}"/*.deb "${PKG_ROOT}/${pkg_name}_${CHANNEL}_${OS}-${ARCH}_${FULLVERSION}.deb"
+    if [[ "$pkg_name" =~ devtools ]]; then
+        BASE_NAME="algorand-devtools"
+    else
+        BASE_NAME=algorand
```
Comment (egieseke) [inline scripts/release/build/deb/package.sh:33]: should this be quoted

File: scripts/release/build/rpm/package.sh [hunk 2]
```diff
@@ -30,9 +30,15 @@ for pkg_name in "${PKG_NAMES[@]}"; do
 
     mkdir "$TEMPDIR/$pkg_name"
 
+    if [[ "$pkg_name" =~ devtools ]]; then
+        INSTALLER_DIR="algorand-devtools"
+    else
+        INSTALLER_DIR=algorand
```
Comment (egieseke) [inline scripts/release/build/rpm/package.sh:36]: quotes

File: scripts/release/mule/package/deb/package.sh [hunk 3]
```diff
@@ -90,8 +90,13 @@ Dpkg::Options {
 EOF
 
 mkdir -p "${PKG_ROOT}/DEBIAN"
+if [[ "$PKG_NAME" =~ devtools ]]; then
+    INSTALLER_DIR="algorand-devtools"
+else
+    INSTALLER_DIR=algorand
```
Comment (egieseke) [inline scripts/release/mule/package/deb/package.sh:96]: quotes

File: scripts/release/mule/package/rpm/package.sh [hunk 4]
```diff
@@ -32,8 +32,13 @@ RPMTMP=$(mktemp -d 2>/dev/null || mktemp -d -t "rpmtmp")
 trap 'rm -rf $RPMTMP' 0
 
 TEMPDIR=$(mktemp -d)
+if [[ "$ALGORAND_PACKAGE_NAME" =~ devtools ]]; then
+    INSTALLER_DIR="algorand-devtools"
+else
+    INSTALLER_DIR=algorand
```
Comment (egieseke) [inline scripts/release/mule/package/rpm/package.sh:38]: quotes

File: scripts/release/build/deb/package.sh (hunk 1 above)
Comment (btoll) [reply scripts/release/build/deb/package.sh:33]: I generally don't quote unless it needs variable expansion or preserving special chars.

**Pull request merged**